docs/architecture/blazor-for-web-forms-developers/project-structure.md

```diff
@@ -246,6 +246,20 @@ To run the Blazor WebAssembly app, choose one of the following approaches:
 
 Blazor WebAssembly apps can be debugged in both browser and Visual Studio.See [Debug ASP.NET Core Blazor WebAssembly](/aspnet/core/blazor/debug) for details.
 
+Both Blazor Server and Blazor WebAssembly apps support Hot Reload in Visual Studio. Hot Reload is a feature that automatically updates changes made to a Blazor app live, in the browser. You can toggle whether Hot Reload is enabled from its icon in the toolbar:
+
+![Hot Reload in Visual Studio 2022](media/project-structure/hot-reload-1.png)
+
+Clicking the caret beside the icon reveals additional options. You can toggle Hot Reload on or off, restart the application, and toggle whether Hot Reload should occur whenever a file is saved.
+
+![Hot Reload menu in Visual Studio 2022](media/project-structure/hot-reload-2.png)
+
+You can also access additional configuration options. The configuration dialog lets you specify whether hot reload should be enabled when debugging (along with Edit and Continue), when starting without debugging, or when a file is saved.
```

Comment:
```suggestion
You can also access additional configuration options. The configuration dialog lets you specify whether Hot Reload should be enabled when debugging (along with Edit and Continue), when starting without debugging, or when a file is saved.
```

You started using it as a proper name and were correctly casing it, but here it was lower case.

Reply:
Missed that in my search, good catch!